packages/v2/README.md

```diff
@@ -0,0 +1,137 @@
+# AO Stamp Protocol v2
+
+Process ID: `DrHOLaZ9E23h6vQuMDl09TciS07yr-qcVHllFICYgAo`
+
+## Getting Started
+
+### Create Process
+
+```sh
+aos <<PROCESS_NAME>> --tag-name Extension --tag-value WeaveDrive --tag-name Variant --tag-value weavedrive.1 --module="_wjbCuCyUTmKAseOfEj0NzrCNUgKcMqBGaSzNySzMoY"  --cron=5-blocks
+```
+
+### Load APM packages
+
+```sh
+.load-blueprint apm
+APM.install('@rakis/WeaveDrive')
+APM.install('@rakis/test-unit')
+APM.install('@tilla/graphql')
+APM.install('@tilla/graphql_server')
+```
+
+### Load Lua code
+
+```sh
+.load current-balances.lua
```

Comment:
Are all these dependencies, you can load these using require functions, or concat into one single lua file to load as well.